packages/server-renderer/__tests__/ssrComputed.spec.ts

```diff
@@ -33,7 +33,6 @@ test('computed reactivity during SSR', async () => {
     // In both cases we need to fetch data.
     if (!msg.value) await store.fetchData()
 
-    expect(msg.value).toBe('hello world')
```

Comment:
For the reasons for deleting this line, see https://github.com/vuejs/core/issues/5300#issuecomment-1790432852